spyder/plugins/editor/widgets/tests/test_introspection.py

```diff
@@ -416,6 +411,19 @@ def test_completions(lsp_codeeditor, qtbot):
 
     assert code_editor.toPlainText() == 'import math\nmath.hypot'
 
+    qtbot.keyPress(code_editor, Qt.Key_Escape)
+
+    try:
+        with qtbot.waitSignal(completion.sig_show_completions,
+                              timeout=10000) as sig:
+            qtbot.keyPress(code_editor, Qt.Key_Tab)
+    except pytestqt.exceptions.TimeoutError:
+        # This one should generate a timeout error because the prefix is the
+        # same that the returned completions of jedi
```

Comment:
```suggestion
        # This should generate a timeout error because the completion
        # prefix is the same that the completions returned by Jedi.
        # This is a regression test for spyder-ide/spyder#11600
```